packages/core/src/framework/plugins/plugin.class.ts

```diff
@@ -1,33 +1,23 @@
-import { CommandApi, CommandResult, CommandService } from '../commands';
-import { EventSource } from '../events';
-
-// @todo: definitions such as this should exist separate from rest of core package.
-//        because plugins extending this class dont need to bring in the rest of core into their bundle.
-//        otherwise we could end up with 2 versions of the core in the final app bundle
+import { AuguryCore } from '../augury-core';
+import { AuguryBridge } from './bridge';
 
 export abstract class Plugin {
-  private commandApi: CommandApi | null = null;
+  protected readonly bridge: AuguryBridge = AuguryBridge.getInstance();
+  protected augury: AuguryCore | null = null;
 
```

Comment:
I'm thinking bridge/core can be merged or the bridge can go into core now.

Reply:
From my perspective the bridge should be an abstract communication channel to any clients of the core. The plugins right now, are kind of more like middleware (imo), so you could be right in the case of this. However, if we wanted to have a history log persistence layer then have plugins do reductions/projections we might want this to be in a different process which may require some sort of bridge channel. Probably makes sense to simplify first then expand when this becomes more clear though.

Reply:
Cool, I will leave it where it is then for now! Thanks!